apps/dsb-client-gateway-api/src/app/modules/message/service/dsb-message-pooling.service.ts

```diff
@@ -163,18 +163,12 @@ export class DsbMessagePoolingService implements OnModuleInit {
 
   public async sendMessagesToSubscribers(messages: GetMessageResponse[], fqcn: string, client: any): Promise<void> {
     try {
-      const emitMode: EventEmitMode = this.configService.get('EVENTS_EMIT_MODE', EventEmitMode.BULK);
-
-      if (emitMode === EventEmitMode.BULK) {
-        const msg = JSON.stringify(messages.map((message) => ({ ...message, fqcn })));
-        this.logger.log(`[WS][sendMessagesToSubscribers][BULK]${msg}`);
-        client.send(msg);
-        return;
-      }
 
       messages.forEach((message: GetMessageResponse) => {
-        this.logger.log(`[WS][sendMessagesToSubscribers][SINGLE]${message}`);
-        client.send(JSON.stringify({ ...message, fqcn }));
+        if (client.readyState === WebSocket.OPEN) {
+          client.send(JSON.stringify({ ...message, fqcn }));
+          this.logger.log(`[WS][sendMessagesToSubscribers][SINGLE] ${JSON.stringify({ ...message, fqcn })}`);
+        }
```

Comment:
Why are we reverting from the Bulk mode?

Reply:
for now to reduce data lost per send